cloudbuild.yaml

```diff
@@ -18,31 +18,31 @@ options:
     name: 'projects/pso-kokoro-resources/locations/us-central1/workerPools/private-pool'
 steps:
 - id: lint
-  name: 'gcr.io/pso-kokoro-resources/python-multi'
+  name: 'gcr.io/pso-kokoro-resources/python-multi-test:staging'
```

Comment:
Have we documented how to (re)build python-multi?

I would ideally like to install Oracle+ in it for example

Reply:
I will build a doc for this, I'm working on building custom images for testing starting with Db2, then moving to oracle and sql server.